src/script/conversation/ConversationRepository.ts

```diff
@@ -249,30 +244,17 @@ export class ConversationRepository {
         }
         if (conversation) {
           const hasChangedLegalHoldStatus = conversation.legalHoldStatus() !== legalHoldStatus;
-          hasNewLegalHoldDevices = newDevices.some(device => device.isLegalHold());
-          if (hasChangedLegalHoldStatus) {
```

Comment:
This was unneeded since the legal hold message is sent via another channel (in the conversation entity)